SU-HSDO/stanford_mrc
# MRCD8-316  MRCD8-322  MRCD8-320  MRCD8-319 MRCD8-325 Feedback updates

# READY FOR REVIEW

# Summary
- Added no result display to featured event block for home page
- made all filters consistent as multiple selects
- added little chevron to chosen select widget
- Limited upcoming events to 2 items.
- Swapped the visitor and news views on the event series page

# Needed By (Date)
- End of sprint

# Urgency
- Low

# Steps to Test

1. checkout branch
2. sync to prod
3. update database
4. ensure the above summary changes are applied.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
